crates/core/src/subscription/module_subscription_actor.rs

```diff
@@ -44,7 +44,7 @@ impl ModuleSubscriptions {
         timer: Instant,
         _assert: Option<AssertTxFn>,
     ) -> Result<(), DBError> {
-        let ctx = ExecutionContext::subscribe(self.relational_db.address());
+        let ctx = ExecutionContext::subscribe(self.relational_db.address(), subscription.query_strings.join("\n"));
```

Comment:
This is why we can't use the `ExecutionContext`. Because we don't want to log all of the queries, just the ones that are slow.